templates/guides/migration-guide-1.0.0.md

```diff
@@ -0,0 +1,79 @@
+- `fastly_service_dynamic_snippet_content_v1` -> `fastly_service_dynamic_snippet_content`
+- `fastly_user_v1` -> `fastly_user`
+
+When switching to the new 1.0.0 release, some existing projects might see an error such as:
+
+```
+Error: no schema available for module.fastly.<...> while reading state; this is a bug in Terraform and should be reported
+```
+
+This error is because the Terraform state file will have a different resource name to what's now exposed via the 1.0.0 release. To resolve this issue you'll need to first remove the old resource from the state file and then reimport the data using the new resource name.
+
+As an example, if you had defined an `fastly_service_acl_entries_v1` resource, then you could remove and reimport using the following commands (any brackets `<...>` should be replaced with actual values):
+
+```bash
+terraform state rm 'fastly_service_acl_entries_v1.<your_resource_name>["<your_acl_name>"]'
+terraform import 'fastly_service_acl_entries.<your_resource_name>["<your_acl_name>"]' <service_id>/<acl_id>
+
+terraform state rm 'fastly_service_v1.<your_resource_name>'
+terraform import 'fastly_service_vcl.<your_resource_name>' <service_id>
+```
+
+It's also recommended that you keep a backup of your state file so you can use it to reference any resource IDs necessary.
+
+**Logging resources have consistent naming format**:
+
+All logging resources now have a consistent naming convention with the provider prefixed with `logging_`.
+
+- `bigquerylogging` -> `logging_bigquery`
+- `blobstoragelogging` -> `logging_blobstorage`
+- `gcslogging` -> `logging_gcs`
+- `httpslogging` -> `logging_https`
+- `logentries` -> `logging_logentries`
+- `papertrail` -> `logging_papertrail`
+- `s3logging` -> `logging_s3`
+- `splunk` -> `logging_splunk`
+- `sumologic` -> `logging_sumologic`
+- `syslog` -> `logging_syslog`
+
+**Director `capacity` removed**:
+
+The Fastly API never supported the `capacity` field for a `director` resource (this was added to the Terraform provider by mistake). Load balancing of director backends is managed by the `weight` field on each associated `backend` resource.
+
+**GCS Logging field `email` renamed**:
+
+The Fastly API was updated with a new [`user`](https://developer.fastly.com/reference/api/logging/gcs/) field to replace `email`.
+
+**Logging `format` and `format_version` defaults changed**:
+
+Pre-1.0.0 the default values for `format` and `format_version` were incorrectly set to an older version `1`. All new logging endpoints use the version `2` custom log format by default.
+
+**Backend `auto_loadbalance` default changed**:
+
+The Fastly web interface defaults "Auto load balance" to "No". The most common reason for having multiple backends in a single service is to route different paths to different backends, rather than load balance between different origins. The provider pre-1.0.0 defaulted `auto_loadbalance` to `true`, which was considered unexpected behaviour. The default is now `false`.
```

Comment:
```suggestion
The Fastly web interface sets "Auto load balance" to "No" by default. The most common reason for having multiple backends in a single service is to route different paths to different backends, rather than load balance traffic between the backends. Conversely, pre-1.0.0, the terraform provider set`auto_loadbalance` to `true` by default, which was inconsistent and often unexpected. The default is now `false`.
```